python/prodV3/worker.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+
+from pathlib import Path
+
+# User packages
+PKGS_PATH = "site/wwwroot/.python_packages"
+VENV_PKGS_PATH = "site/wwwroot/worker_venv"
+
+PKGS_36 = "lib/python3.6/site-packages"
+PKGS = "lib/site-packages"
+
+# Azure environment variables
+AZURE_WEBSITE_INSTANCE_ID = "WEBSITE_INSTANCE_ID"
+AZURE_CONTAINER_NAME = "CONTAINER_NAME"
+
+
+def is_azure_environment():
+    return (AZURE_CONTAINER_NAME in os.environ
+            or AZURE_WEBSITE_INSTANCE_ID in os.environ)
+
+
+def determine_user_pkg_paths():
+    minor_version = sys.version_info[1]
+
+    home = Path.home()
+    pkgs_path = os.path.join(home, PKGS_PATH)
+    venv_pkgs_path = os.path.join(home, VENV_PKGS_PATH)
+
+    user_pkg_paths = []
+    if minor_version == 6:
+        user_pkg_paths.append(os.path.join(venv_pkgs_path, PKGS_36))
+        user_pkg_paths.append(os.path.join(pkgs_path, PKGS_36))
+        user_pkg_paths.append(os.path.join(pkgs_path, PKGS))
+    elif minor_version == 7:
+        user_pkg_paths.append(os.path.join(pkgs_path, PKGS))
```

Comment:
Should also include a check here for 3.8: `if minor_version == 7 or minor_version == 8`